frontend/lib/norent/letter-content.tsx

```diff
@@ -76,13 +100,15 @@ function friendlyUTCDate(date: GraphQLDate) {
   return friendlyDate(new Date(date), "UTC");
 }
 
-const PaymentDate: React.FC<{ paymentDate: GraphQLDate }> = (props) => (
-  <>{friendlyUTCDate(props.paymentDate)}</>
+const PaymentDate = componentizeHelper((props) =>
+  friendlyUTCDate(props.paymentDate)
 );
 
 const LetterHeading: React.FC<NorentLetterContentProps> = (props) => (
   <dl className="jf-letter-heading">
-    <dt>To</dt>
+    <dt>
+      <Trans description="before address in formal letter">To</Trans>
+    </dt>
```

Comment:
Hmmm, now that I read it again, I'm wondering if we could group the whole ```To: Landlordio ...... From: Tenant ......``` section as one translation? I know that there would be a lot of variables inside the translation text but perhaps those could actually help give some context to the translator. These floating "To" and "From" snippets give me a bit of pause, as different languages may have a different structure for letter heads. Up to you though!

Reply:
Oops I replied to this comment over here: https://github.com/JustFixNYC/tenants2/pull/1496#discussion_r431892451